tests/test_e2e.py

```diff
@@ -0,0 +1,72 @@
+from sklearn import datasets
+from sklearn.cluster import KMeans
+from sklearn.linear_model import (
+    LinearRegression,
+    LogisticRegression,
+)
+from sklearn.model_selection import train_test_split
+
+try:
+    # neptune-client=0.9.0 package structure
+    import neptune.new as neptune
+except ImportError:
+    # neptune-client=1.0.0 package structure
+    import neptune
+
+import neptune_sklearn as npt_utils
+
+
+def test_classifier_summary():
+    run = neptune.init_run()
+
+    iris = datasets.load_iris()
+    X = iris.data[:, :2]
+    y = iris.target
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5)
+
+    model = LogisticRegression(C=1e5)
+    model.fit(X_train, y_train)
+
+    run["summary"] = npt_utils.create_classifier_summary(model, X_train, X_test, y_train, y_test)
+
+    run.wait()
+    validate_run(run, log_charts=True)
```

Comment:
Should we also test with `log_charts=False`?

Reply:
The tests are right now very minimalistic, basically a smoke test. Definitely, it'd benefit to improve them, but I'm not sure if this is in scope.

Reply:
Fair enough.